testing-common/src/main/groovy/io/opentelemetry/auto/test/base/HttpServerTest.groovy

```diff
@@ -453,7 +453,7 @@ abstract class HttpServerTest<SERVER> extends AgentTestRunner {
   }
 
   // parent span must be cast otherwise it breaks debugging classloading (junit loads it early)
-  void serverSpan(TraceAssert trace, int index, String traceID = null, String parentID = null, String method = "GET", ServerEndpoint endpoint = SUCCESS) {
+  void serverSpan(TraceAssert trace, int index, String traceID = null, String parentID = null, String method = "GET", ServerEndpoint endpoint = SUCCESS, Response response = null) {
```

Comment:
can you change this to pass in `int responseContentLength` instead of the okhttp `Response`, and also swap the last two params, so then can leave off `endpoint` from the callers since most don't use it?

Reply:
sure

Reply:
I'll need to make it a nullable object for the optional parameters to work, and also a Long to remove unnecessary type casting. Perhaps the SemanticAttributes.HTTP_RESPONSE_CONTENT_LENGTH should have a long type instead of string as well.

Reply:
👍 I opened https://github.com/open-telemetry/opentelemetry-java/issues/1442 this morning while reviewing your PR